src/scripts/fetch_package_names.js

```diff
@@ -65,3 +105,98 @@ function fetchPackageNames(options, cb) {
     }
   );
 }
+
+function fetchPackageFiles(options, finalCb) {
+  const file = options.file;
+
+  async.mapLimit(
+    options.input,
+    options.limit,
+    function(pkg, cb) {
+      const branch = 'master';
+      const url = ['https://raw.githubusercontent.com', pkg.full_name, branch, file].join('/');
+
+      request(url, function(err, response, body) {
+        if (err) {
+          return cb(err);
+        }
+
+        if (body && file === 'README.md') {
```

Comment:
I think we had this in other PR but is it safe to assume that all the newly added webpack-contrib package readme files will be named exactly `README.md` and not `readme.md` or `README.MD` etc?

Reply:
No